kit/internal/rebuilder/writer.go

```diff
@@ -0,0 +1,53 @@
+package rebuilder
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+	"strings"
+	"time"
+)
+
+var maxServiceNameLen = 4
+
+var colors = []string{
+	"\033[92m", // Green
```

Comment:
This needs to be unlimited, maybe we cycle the services/processes once the limit is reached. We could also have a reasonably higher limit @tatang26.

Reply:
Overall its looking great @tatang26 !